merlin-driver/src/main/java/gov/nasa/jpl/aerie/merlin/driver/MissionModel.java

```diff
@@ -69,7 +69,7 @@ public Task getDaemon() {
       @Override
       public TaskStatus step(final Scheduler scheduler) {
         MissionModel.this.daemons.forEach(daemon -> scheduler.spawn(daemon.create()));
-        return TaskStatus.completed();
+        return TaskStatus.completed(null);
```

Comment:
Why `null` and not `VoidEnum.VOID`?

Reply:
I moved VoidEnum.VOID into merlin-framework, where it's not accessible from the driver. Do you figure it would be worth moving it back into protocol? Or maybe defining a similar Unit enum internally for the driver's use? 🤔

Reply:
Good point, but yes, I'd avoid `null` wherever possible.

To be honest, I'm of the opinion that combining all of the daemons into one top-level task was actually a mistake born of convenience. Each daemon will automatically have its own return type anyway, and there's no good reason for us to group them up like this.

And `ControlTask` (the other place this happens) is already gone on one of my branches.

So yeah, I'd recommend adding a local `VOID` for the sake of avoiding `null`. No reason everybody has to use the same `VOID`.

Reply:
I added a local Unit :+1: The only `completed(null)` calls left are:
- ReplayingTask status field inits to `TaskStatus.completed(null)`
- ThreadedTask aborts with a `completed(null)`

These have to be null, since they must be values of the `Return` type, but the Supplier has not returned any value.

Future work: handle ThreadedTask aborting with a different variant of either TaskStatus or TaskResponse.